cmd/snap/cmd_list.go

```diff
@@ -75,7 +75,7 @@ func listSnaps(names []string) error {
 			Private:  snap.Private,
 			DevMode:  snap.DevMode,
 			TryMode:  snap.TryMode,
-			Inactive: snap.Status == client.StatusInstalled,
+			Disabled: snap.Status == client.StatusInstalled,
```

Comment:
Hmm? so Disabled will be true when snap.Status is client.StatusInstalled? I must be missing something but this feels like it makes no sense.